### ISSUE TITLE
Aerobic Respiration

### ISSUE THREAD

**Issue by ResetPower** [ResetPower/ResetPower#7]
# Aerobic Respiration

The process of aerobic respiration is quite complex. It can be briefly summarized into three stages.

The summary formula of aerobic respiration is $\mathrm{C_6H_{12}O_6+6H_2O+6O_2\overset{Enzyme}{-\rightarrow}6CO_2+12H_2O+Energy}$.

## First stage

First stage happens in the **cytoplasmic matrix**, 1 molecule of glucose ( $\mathrm{C_6H_{12}O_6}$ ) is decomposed into 2 molecules of pyruvic acid ( $\mathrm{C_3H_4O_3}$ ) and a few $\mathrm{[H]}$.

> Note that the process of producing $\mathrm{[H]}$ actually means to transfer **oxidized coenzyme I** ( $\mathrm{NAD^+}$ ) to **reduced coenzyme I** ( $\mathrm{NADH}$ ).

In conclusion, $\mathrm{C_6H_{12}O_6\overset{Enzyme}{-\rightarrow}2C_3H_4O_3+4[H]}$.

## Second stage

Second stage happens in the **mitochondrial matrix**, pyruvic acid and water are decomposed into carbon oxide and [H], which is $\mathrm{2C_3H_4O_3+6H_2O\overset{Enzyme}{-\rightarrow}6CO_2+20[H]}$.

## Third stage

Third stage happens in the **inner mitochondrial membrane**, [H] produces from the previous two stages are transferred into water, this stage needs oxygen and generated plenty of ATP(Energy), which is $\mathrm{24[H]+6O_2\overset{Enzyme}{-\rightarrow}12H_2O}$.
